Workflow/Tests/ConcurrencyTests.swift

```diff
@@ -160,9 +160,8 @@ final class ConcurrencyTests: XCTestCase {
         let secondScreen = host.rendering.value
         XCTAssertEqual(1, secondScreen.count)
 
-        // MANUAL TEST CASE: Uncomment to validate this fatal errors.
-        // Calling `update` uses the original sink. This will fail with a fatalError as the sink was not redeclared.
-        // initialScreen.update()
+        // Calling `update` uses the original sink.
+        initialScreen.update()
```

Comment:
it's because the internal 'reusable sink' that forwards the event through the 'event pipe' has already been deallocated. i believe i inadvertently made the commented-out test no longer crash with [this change](https://github.com/square/workflow-swift/pull/189), though at the time i was unaware there were any tests for this scenario.

Reply:
i will update the comment